src/Components/News/NewsListItem.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Media } from 'reactstrap';
+import { Card, CardImg, CardTitle, CardText, CardSubtitle, CardBody } from 'reactstrap';
 import { Link } from 'react-router-dom';
+import DateFormat from '../Shared/DateFormat';
+import '../../theme/css/card.css';
```

Comment:
We should put this in the theme level of css, so we don't have to import `card.css` in each component we want to use it.

Here I'm not sure if is a good idea to use css native imports because it would do an extra request for each css we import.